### ISSUE TITLE
(Improvement)  Reorganizing the app tabs into categories 

### ISSUE THREAD

**Issue by fux0r2009** [ferraridamiano/ConverterNOW#31]
Hi @ferraridamiano  I would like to make a proposition for the organization of the app as I believe it would be better if we have **Master Categories** instead of having all the tabs on the side of the app, I propose to use categories like: 

**Geometry:** Length, Area, Volume, Angles
**Finance :** Currencies
**Physics:** Force, Speed, Temperature, Time Energy, Power, Pressure, Mass
**Numerical:** Numeral systems, SI Prefix
**Practical:** Fuel Consumption, Shoe size, Digital Data, Torque 

I think It will make the app more organized and easier quicker to navigate!

hope you give it a thought :)

**Comment by ferraridamiano** [comment]
Hi @fux0r2009 and thanks for your contribution! This could be part of a more structured UI improvement. I don't know if you are good with graphics and mockup, but could you suggest some new app structures with a mockup/screenshot example?

Thanks!

**Comment by fux0r2009** [comment]
@ferraridamiano Thank you for your feedback, Yes of course I am not very good with graphic design but here is a possible mockup of the ideas I suggested earlier : 

<a href="https://ibb.co/0FWJD1g"><img src="https://i.ibb.co/KLQy0BM/Mockup.png" alt="Mockup" border="0"></a><br />

Of course the menu buttons at the top are like a sliding ribbon in which you can add as many categories as you want... and within each category you can have a menu with all the corresponding conversions to keep things organized!

Regards :)

**Comment by ferraridamiano** [comment]
Thanks, I will consider your proposal in a future app restyle

**Comment by fux0r2009** [comment]
Hi @ferraridamiano another example I found of very good UI can be found in this app: 

https://play.google.com/store/apps/details?id=com.eclix.unit.converter.calculator

With black theme It would look fantastic!

Best regards

**Comment by ferraridamiano** [comment]
Thanks! I will consider different solutions. The only constrain is that the future UI should be at least immediate and simple as it is now

**Comment by dmint789** [comment]
I would also like to request an option to reorder the different units under each category. This is especially annoying with currencies, as there are so many of them, so I always have to scroll for a while to find the two I want to convert between.

**Comment by ferraridamiano** [comment]
@dmint789, reordering units and properties is already available in the current version (just go to settings).
What is your suggestion for currencies? A division by continent of the respective nation?

**Comment by dmint789** [comment]
No, I meant the sub-units in each category. Let's say I use inches and miles more than meters, then I'd move them to the top. And then I'd rearrange currencies to have us dollars and british pounds at the top, so that I don't have to scroll every time.
